tests/conftest.py

```diff
@@ -249,3 +251,45 @@ def persisted_channel(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="package")
+def grpc_session_creation_kwargs():
+    """Gets the keyword arguments required for creating the gRPC interpreter."""
+    channel = grpc.insecure_channel(f"localhost:31763")
```

Comment:
Please extract this into a separate `grpc_channel` fixture. 

Is scope=package the right scope? I suspect that scope=package would re-create the fixture for each directory. Maybe we want scope=session.

Use a context manager to ensure that it closes the channel properly. The scope makes this less important, but it's still a good practice. 
```
    with grpc.insecure_channel("localhost:31763") as channel:
        yield channel
```

The correct return type for a fixture that fields a `grpc.Channel` is `Generator[grpc.Channel, None, None]`.

Reply:
Updated as per suggestion, i have updated the scope to session